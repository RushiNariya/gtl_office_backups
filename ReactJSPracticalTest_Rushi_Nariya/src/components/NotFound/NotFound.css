.not-found {
  margin: auto !important;
  padding-left: 100px;
}
