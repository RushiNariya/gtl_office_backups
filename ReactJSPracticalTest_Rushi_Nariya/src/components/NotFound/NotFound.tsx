import React from 'react';
import './NotFound.css';

function NotFound(): JSX.Element {
  return (
    <>
      <article className="city">
        <div className="column-1 not-found">No cities Found!</div>
      </article>
    </>
  );
}

export default NotFound;
