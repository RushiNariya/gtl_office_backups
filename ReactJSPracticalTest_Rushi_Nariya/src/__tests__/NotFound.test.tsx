import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import NotFound from '../components/NotFound/NotFound';

test('renders the NotFound Component', (): void => {
  const component: ShallowWrapper = shallow(<NotFound />);
  expect(component).toMatchSnapshot();
});
