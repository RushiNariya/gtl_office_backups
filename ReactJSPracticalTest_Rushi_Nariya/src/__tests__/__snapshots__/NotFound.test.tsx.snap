// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the NotFound Component 1`] = `
<Fragment>
  <article
    className="city"
  >
    <div
      className="column-1 not-found"
    >
      No cities Found!
    </div>
  </article>
</Fragment>
`;

exports[`renders the component 1`] = `
<Fragment>
  <article
    className="city"
  >
    <div
      className="column-1 not-found"
    >
      No cities Found!
    </div>
  </article>
</Fragment>
`;
